betty/extension/cotton_candy/webpack/main.ts

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+import { initializeFiles } from'./file.ts'
+import { Search } from './search.ts'
+import { initializeToggles } from './show.ts'
+import './main.scss'
+
+async function main(): Promise<void> {
+    initializeFiles()
+    initializeToggles()
+    const search = new Search()
+    await search.initialize()
+}
+void main()
```

Comment:
Can we use the void operator elsewhere to replace ESLint ignores for misused promises?

Reply:
Fixed by https://github.com/bartfeenstra/betty/pull/1435